db.tf

```diff
@@ -68,3 +69,37 @@ resource "aws_route53_record" "mysql-shared-dns" {
   ttl = 300
   records = ["${aws_db_instance.mysql-shared-db.address}"]
 }
+
+resource "aws_db_instance" "postgres-shared-db" {
+    allocated_storage         = 300
+    engine                    = "postgres"
```

Comment:
We should also specify `engine_version` so we know which version of postgres we're going to get. Amazon recently launched support for 9.5 so we should go with the latest version of that (9.5.4)